templates/user/auth/signin_inner.tmpl

```diff
@@ -19,7 +19,9 @@
 			{{if or (not .DisablePassword) .LinkAccountMode}}
 			<div class="required field {{if and (.Err_Password) (or (not .LinkAccountMode) (and .LinkAccountMode .LinkAccountModeSignIn))}}error{{end}} form-field-content-aside-label">
 				<label for="password">{{ctx.Locale.Tr "password"}}</label>
-				<a href="{{AppSubUrl}}/user/forgot_password">{{ctx.Locale.Tr "auth.forgot_password"}}</a>
+				<div>
+					<a href="{{AppSubUrl}}/user/forgot_password" tabindex="-1">{{ctx.Locale.Tr "auth.forgot_password"}}</a>
+				</div>
```

Comment:
Why a link should be inaccessible by the keyboard?

It breaks the keyboard-users and screen readers.

Reply:
Yes, I found the `tabindex` too but don't think it's a good solution. Other [logins](https://github.com/login) (Github, IMDb) uses the same structure but the link does not get the focus without a `tabindex`.

Reply:
The [IMDb](https://www.imdb.com/ap/signin?openid.pape.max_auth_age=0&openid.return_to=https%3A%2F%2Fwww.imdb.com%2Fregistration%2Fap-signin-handler%2Fimdb_us&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.assoc_handle=imdb_us&openid.mode=checkid_setup&siteState=eyJvcGVuaWQuYXNzb2NfaGFuZGxlIjoiaW1kYl91cyIsInJlZGlyZWN0VG8iOiJodHRwczovL3d3dy5pbWRiLmNvbS8_cmVmXz1sb2dpbiJ9&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&tag=imdbtag_reg-20) login doesn't have a `position: absolute`. Can't see what trick they use.

Reply:
Interesting. By natural HTML order, the link should focus first there, but does not. Can't see anything immediately obvious that would influence the tabbing order either.

Reply:
Actually I see the imdb page is just using manual `tabindex`.

1. Username
2. Password
3. Sign In Button

I guess in our case we have to insert "Remember me" in position 3.

Reply:
Done that, so for us it's 

1. Username
2. Password
3. Rember Me
4. Sign In Button

Reply:
Tweaked once more:

1. Username
2. Password
3. Sign In
4. Forgot Password
5. Rember Me

I think it's good to optimize the common case of username,password,signin. We are lacking a focus style on buttons in general which is a accessibility concern, but we can fix that another time.